tasks/libs/common/github_api.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+import platform
+import subprocess
+
+from invoke.exceptions import Exit
+
+from .remote_api import RemoteAPI
+
+__all__ = ["GithubAPI"]
+
+
+class GithubAPI(RemoteAPI):
```

Comment:
Here and in the `GithubWorkflows` class everything takes a `repo_name` parameter. Should we make that a class  attribute instead?

Reply:
Done!